use_cases/eluc/data/eluc_data.py

```diff
@@ -138,21 +138,6 @@ def hf_to_df(self, hf_repo):
         df = df.set_index(["time", "lat", "lon"], drop=False)
         return df
 
-      
```

Comment:
Made a small mistake with the merge resolver. I don't think this actually breaks anything though thankfully